javascript-light-bulb/main.js

```diff
@@ -0,0 +1,19 @@
+var $light = document.querySelector('.circle');
+var $background = document.querySelector('.circle-container');
+var clicks = 0;
+
+$light.addEventListener('click', onOff);
+
+function onOff(event) {
+  if (clicks % 2 > 0) {
```

Comment:
Instead of using a counter variable and just checking if the current count is even or odd, there is a better data type that has only two states that you can use to determine if the light is "on" or not: a boolean.